omx/gstomxaudiodec.c

```diff
@@ -917,21 +909,38 @@ gst_omx_audio_dec_set_format (GstAudioDecoder * decoder, GstCaps * caps)
       return FALSE;
   } else {
     if (!(klass->cdata.hacks & GST_OMX_HACK_NO_DISABLE_OUTPORT)) {
-      /* Disable output port */
-      if (gst_omx_port_set_enabled (self->dec_out_port, FALSE) != OMX_ErrorNone)
+
+      GST_DEBUG_OBJECT (self, "Enabling ports");
+
+      /* Enable input port */
+      if (gst_omx_port_set_enabled (self->dec_in_port, TRUE) != OMX_ErrorNone)
```

Comment:
If you are not returning an error code, and if the method can fail in several points, I recommend adding error messages.

Reply:
Checking the methods you pointed out,  they are already using the GST_DEBUG_ resource to report errors and warnings, this is done internally in the gstomx core file. So adding extra external may add unnecessary noise.